README.md

```diff
@@ -13,9 +13,24 @@ Plugin to add the OpenwebConcept OpenPDC to your project.
 ### How do I get set up?
 
 -   Unzip and/or move all files to the /wp-content/plugins/pdc-base directory
--   Log into WordPress admin and activate the ‘PDC Base’ plugin through the ‘Plugins’ menu
+-   Log into WordPress admin and activate the ‘PDC Base’ plugin through the ‘Plugins’ menu**
```

Comment:
Waar dienen de 2 asterisks voor op het eind?

Reply:
De idee was/is is om te verwijzen naar de corresponderende **Error(s)?** eronder.